portfolio/src/main/webapp/script.js

```diff
@@ -12,17 +12,66 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-/**
- * Adds a random greeting to the page.
- */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
-
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
-
-  // Add it to the page.
-  const greetingContainer = document.getElementById('greeting-container');
-  greetingContainer.innerText = greeting;
+
+var i = 0; 			// Start Point
+var images = [];	// Images Array  
+var time = 3000;	// Time Between Switch
+	 
+// Image List
+images[0] = "images/skate2.jpg";
+images[1] = "images/skateboard.jpg";
+
+images[2] = "images/read.jpg";
+images[3] = "images/read3.jpg";
+images[4] = "images/read4.jpg";
+
+images[5] = "images/photography1.jpg";
+images[6] = "images/photography-1.jpg";
+images[7] = "images/photography.jpg";
+images[8] = "images/photo-3.jpg";
+images[9] = "images/photo2.jpg";
+
+images[10] = "images/art.jpg";
+images[11] = "images/art4.jpg";
+images[12] = "images/art5.jpg";
+
+
```

Comment:
Descriptive comments would be helpful to understand the functionality

Reply:
Noted, I will add descriptive comments.